src/helper/args/mod.rs

```diff
@@ -47,7 +47,9 @@ fn check_args<'a, ArgsIter: Iterator<Item = &'a str>, Output: Write>(
                 "success".cyan().bold(),
                 arg.white().italic()
             )?;
+            println!("---");
             output.write_all(&cmd_out.stdout)?;
+            println!("---");
```

Comment:
Let's do the same thing for line 71 and 75 :3

Reply:
Updated! :)

Reply:
Oh, and I forgot to say that you should use `writeln!(output, "---")?;` instead of `println` for printing the separators to the correct output. Can you update the PR according to that?

Reply:
Ah gotcha. I've replaced the `println` with `writeln!(output, "---")?;` now. Updated the tests as well.